### ISSUE TITLE
Export-NavContainerDatabasesAsBacpac downloading Dac Framework 18.2

### ISSUE THREAD

**Issue by svwfr** [microsoft/navcontainerhelper#510]
When I run Export-NavContainerDatabasesAsBacpac -containerName BCs1gb -bacpacFolder "C:\ProgramData\NavContainerHelper\Extensions\BCs1gb\my", after logged in to the service i got this error:
Downloading Dac Framework 18.2
Exception calling "DownloadFile" with "2" argument(s): "The remote name could not be resolved: 'download.microsoft.com'"
at InstallPrerequisite, <No file>: line 20
at Install-DACFx, <No file>: line 30
at <ScriptBlock>, <No file>: line 206
Exception calling "DownloadFile" with "2" argument(s): "The remote name could not be resolved: 'download.microsoft.com'"
At C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.6.2.4\ContainerHandling\Invoke-ScriptInNavContainer.ps1:30 char:13
+             Invoke-Command -Session $session -ScriptBlock $scriptbloc ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [InstallPrerequisite], MethodInvocationException
    + FullyQualifiedErrorId : WebException,InstallPrerequisite
    + PSComputerName        : f00074cxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

I have updated navcontainer helper
Installed manually Dac Framework 18.2 and find the SqlPackage.exe under folder: C:\Program Files\Microsoft SQL Server\150\DAC\bin

why do i get this error?

**Comment by freddydk** [comment]
When you started the container you probably also got this warning:
WARNING: DNS resolution not working from within the container.
This means that your docker installation for some reason (probably your host network configuration) doesn't allow the container to use the host as DNS.
One solution could be to add a DNS entry to your Docker Settings -> Daemon -> Advanced configuration:
example:
```
{
  "registry-mirrors": [],
  "insecure-registries": [],
  "debug": true,
  "experimental": false,
  "dns": [
    "8.8.8.8"
  ]
}
```
8.8.8.8 is google's public DNS server - you can also use the same DNS as on your host machine (ipconfig /all)

**Comment by freddydk** [comment]
In the next version of navcontainerhelper, you will also be able to specify a dns server to new-navcontainer.

**Comment by freddydk** [comment]
Shipped in 0.6.2.8

**Comment by svwfr** [comment]
Thx Freddy,
You had right with your statement that I earlier had the DNS-Waring, and at that point didn't understand the impact of the warning.
I updated navcontainerhelper to the latest version: 0.6.2.91 and added "dns 8.8.8.8" when create new navcontainer. 
Now I successfully create the bacpac file by use command: Export-NavContainerDatabasesAsBacpac
